src/customJs/tools/promo_code/PromoCodeViewer.tsx

```diff
@@ -11,8 +11,16 @@ export const PromoCodeViewer: ViewerComponent<PromoCodeValues> = ({
   return values.promo_code === EMPTY_SELECTION ? (
     <EmptyViewer {...rest} />
   ) : (
-    <code>
-      <pre>{JSON.stringify(values, undefined, 2)}</pre>
-    </code>
+    <div
+      style={{
+        textAlign: values.alignment,
+        backgroundColor: values.backgroundColor,
+      }}
+      role="container"
+    >
+      <span style={{ color: values.textColor, fontSize: values.fontSize }}>
```

Comment:
At this stage, `backgroundColor`, `textColor` and `fontSize` are undefined, right?

Does it generate a valid HTML when these values are undefined?

Reply:
that right is a valid HTML without style, at the next PR I will add the property to show the styles on the view